genie-web/src/main/java/com/netflix/genie/web/services/impl/LocalJobKillServiceImpl.java

```diff
@@ -206,4 +249,8 @@ private void killJobOnUnix(final int pid) throws GenieException {
             throw new GenieServerException("Unable to kill process " + pid, ioe);
         }
     }
+
+    private boolean isV4Job(final String id) {
```

Comment:
this method seems useless. Just inline the call the the persistence service.

Reply:
Done